xml/System.Globalization/CultureInfo.xml

```diff
@@ -1208,7 +1208,7 @@ If `name` is <xref:System.String.Empty?displayProperty=nameWithType>, the constr
   
      For more information, see the "Culture and task-based asynchronous operations" section in the <xref:System.Globalization.CultureInfo> topic.  
   
--   By calling the Windows `GetUserDefaultLocaleName` function.  
+-   By calling the `GetUserDefaultLocaleName` function on Windows or the `uloc_getDefault` function from [ICU](https://icu-project.org/), which currently calls the POSIX `setlocale` function with category `LC_MESSAGES`, on Unix-like systems.  
```

Comment:
@carlossanlop could you please double check if we use the md hyperlinks styles in our docs as it is used here in the ICU link?

Reply:
Yes, the way it was used in this change is correct. You can also confirm that it looks good if you go to the preview URL: https://review.docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.currentculture?view=dotnet-uwp-10.0&branch=pr-en-us-5071#how-a-threads-culture-is-determined

Scroll a bit down after the code snippet.